application/common/models/User.php

```diff
@@ -149,6 +155,26 @@ public function beforeDelete()
         return true;
     }
 
+    // The MfaBackend code for deleting a webauthn type MFA
```

Comment:
I don't understand this comment and why it's relevant. It sounds like justification for doing things the "hard" way or doing it here instead of in the MfaBackend code. But this implementation looks about as simple as it could be and doesn't look out of place to me.